# This is a message to hold the name of the current state of the finite state machine
Header header
string fsm_state
